addons/payment/static/src/js/express_checkout_form.js

```diff
@@ -13,6 +13,7 @@ publicWidget.registry.PaymentExpressCheckoutForm = publicWidget.Widget.extend({
         await this._super(...arguments);
         this.txContext = {};
         Object.assign(this.txContext, this.$el.data());
+        this.txContext.shippingInfoRequired = !!this.txContext.shippingInfoRequired;
```

Comment:
ES2019 restriction has been lifted 3 months too late :smiling_face_with_tear: 
`= this.txContextshippingInfoRequired ?? false`